backend/server.js

```diff
@@ -18,14 +16,23 @@ app.post('/', async (req,res) => {
     res.send(req.body.text) //store content under category "text"
 })
 
+app.get('/separateContent', (req,res) => {
+    res.send(req.body.content)
+})
+
+app.get('/separateAuthor', (req,res) => {
+    res.send(req.body.author)
+})
+
 
 app.post('/posttoforum', async (req, res) => {
     try {
-        await client.db("storage").collection("posts").insertOne(req.body.content)
-        await client.db("storage").collection("authors").insertOne(req.body.author)
-        res.status(200).send("Database has stored the post\n")
-        //const result =  client.db("data").collection("forum")//.find(req.body)
-        //res.send(result)
+        await client.db("storage").collection("posts").insertOne(req.body)
+        //await client.db("storage").collection("authors").insertOne(req.body.author)
+        const result =  client.db("data").collection("forum").find(req.body)
```

Comment:
Codacy has a fix for the issue: [Replace `····const·result·=··client.db("data").collection("forum").find(req.body)` with `const·result·=·client.db("data").collection("forum").find(req.body);`](https://app.codacy.com/gh/karatwan/CPEN321_world_resources/pullRequest?prid=9811192)
```suggestion
    const result = client.db("data").collection("forum").find(req.body);
```